src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -14,6 +14,7 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        return (a & b) & !c & !d | (a & c) & !b & !d | (a & d) & !c & !b | (c & d) & !a & !b
```

Comment:
Why have you decided to use & and | instead of && and ||  ? 
https://www.tutorialspoint.com/Differences-between-and-and-and-and-operators-in-Java
and this solution might be a bit shorter, try to use ^ (XOR) here, and see if we can simplify it

Reply:
done. 
thanks for advice.